_editing/4_editable_regions.md

```diff
@@ -46,3 +46,13 @@ Gives options for bold, italics and links
 On block elements you can set headings, bold, italics, links, quotes, lists and images.
 
 ![Block Elements](/img/editing/7.png)
+
+### Note about editable class in _includes
+
+If you add a class "editable" to an element in a file in _includes directory and then include it with 
+{% highlight liquid %}
+{% raw %}
+{% include your_file.html %}
+{% endraw %}
+{% endhighlight %}
+You will not be able to edit it in the Visual Editor (in Cloudcannon).
```

Comment:
Yes I know, that's why I think that it's a good idea to say that in the documentation.
If you prefer, you can just say that it's not supported.
But if you clarify that in the documentation, it will save people's time.